From 07a91984b37a380a933bb0f7a21a3c3698cc8f51 Mon Sep 17 00:00:00 2001
From: "Chen, Gang G" <gang.g.chen@intel.com>
Date: Mon, 11 Dec 2023 21:09:37 +0800
Subject: [PATCH 35/54] Remove bootloader partition hash calculation for SBL

SBL only loads and verifies kf4sbl.elf file, it does not calulate
the whole partition

Tracked-On: OAM-113633
Signed-off-by: Chen, Gang G <gang.g.chen@intel.com>
---
 libfastboot/fastboot_oem.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libfastboot/fastboot_oem.c b/libfastboot/fastboot_oem.c
index b3b5734..3d38fea 100644
--- a/libfastboot/fastboot_oem.c
+++ b/libfastboot/fastboot_oem.c
@@ -248,7 +248,9 @@ static struct oem_hash {
 #endif
 	{ TOS_LABEL,		get_boot_image_hash,	TRUE },
 #endif
+#ifndef USE_SBL
 	{ BOOTLOADER_LABEL,	get_bootloader_hash,	FALSE },
+#endif
 	{ VBMETA_LABEL,		get_vbmeta_image_hash,	FALSE },
 #ifdef DYNAMIC_PARTITIONS
 	{ SUPER_LABEL,		get_super_image_hash,	TRUE }
-- 
2.25.1

